apps/davi/src/stores/index.tsx

```diff
@@ -68,8 +68,9 @@ export const HookStoreProvider: React.FC<
       if (!bytecodeHash) {
         const bytecode = await provider.getCode(daoId);
         bytecodeHash = Web3.utils.keccak256(bytecode);
-
-        localStorage.setItem(`hashed-bytecode-${daoId}`, bytecodeHash);
+        if (!bytecodeHash) {

```

Comment:
looks like redundant but the first if is checking if it has to save it to localstorage and the second one if the one obtained is null. 
yes, i it would be the same to check if the bytecode isn't `null` or `undefined` , i can change it for that

Reply:
This looks like if the bytecodeHash is `null`, it'll save it to localStorage. Shouldn't it be `if(bytecodeHash) { save }`?

Also, agree on checking for daoId `null` and prevent saving a `null` bytecodeHash.

Renaming the bytecodeHash variables might help with readability. Maybe `localBytecodeHash` and `fetchedBytecodeHash`?

Reply:
thanks for your comments! i think i addressed them all. let me know otherwise